Documentation/ceph-recover-pod-on-LostNode.md

```diff
@@ -0,0 +1,138 @@
+---
+title: ceph-recover-pod-on-LostNode
+weight: 11125
+indent: true
+---
+
+# Node lost and still has the pod in it
+
+### In this case, you will see one of the pod is **stuck** in the `Terminating` state and another pod in the `ContainerCreating` state.
+
+```console
+kubectl -n rook-ceph get pod
+```
+>```
+>csi-rbd-demo-depl-69cd495f9b-8ldf9                  0/1     ContainerCreating   0          24m
+>csi-rbd-demo-depl-69cd495f9b-nl6hf                  1/1     Terminating         0          32m
+>```
+
+And, when you describe the pod stuck in `ContainerCreating` you see `Warning` like below
+
+```console
+$ kubectl -n rook-ceph describe pod csi-rbd-demo-depl-69cd495f9b-8ldf9
+
+Warning  FailedAttachVolume  17m                  attachdetach-controller                Multi-Attach error for volume "pvc-ccb1108e-4d3d-4478-9c16-56a1e3001a7a" Volume is already used by pod(s) csi-rbd-demo-depl-69cd495f9b-rwdxq
+Warning  FailedMount         3m43s (x6 over 15m)  kubelet, ip-10-0-159-129.ec2.internal  Unable to attach or mount volumes: unmounted volumes=[mypvc], unattached volumes=[mypvc default-token-s2fdk]: timed out waiting for the condition
+```
+
+**Solution:**
+
+Delete the pod stuck in `Terminating` state forcefully
+```console
+$ kubectl -n rook-ceph delete pod csi-rbd-demo-depl-69cd495f9b-nl6hf --grace-period 0 --force
+```
+
+**And** then wait for few minutes.
```

Comment:
How long did you wait? When this issue hit Seb and I a few days ago, waiting a few minutes did not help to get it started. Perhaps we should combine the next section with this one so it will be clear that additional steps may need to be taken to remove the watcher.

Reply:
we don't need to wait as we have deleted the pod forcefully we just need to ensure the pod is gone from the Node which is lost.

Reply:
@Madhu-1 IIUC after deleting the pod forcefully still have to wait around 5 to 7 min for container creating to be in running state.

Reply:
that depends on the manual recovery steps we do. but AFAIC this waiting is to make sure the pod gets deleted which is in a terminating state.

Reply:
Thanks @Madhu-1 for clearing the confusion. I'll remove this `And then wait...`

Reply:
To be clear, there are two parts to deleting the pod. 
1. Force delete the pod. K8s will remove the pod from the API server after some short timeout. (a minute or two?)
2. If the node is lost (network is not connecting) the pod could actually remain running on that node. The admin must confirm that the node will never come back or else it could cause corruption.

We need to document both of the steps.